homeassistant/components/google_assistant/trait.py

```diff
@@ -648,7 +648,14 @@ async def _execute_cover(self, command, data, params, challenge):
         """Execute a StartStop command."""
         if command == COMMAND_STARTSTOP:
             if params["start"] is False:
-                if self.state.state in (cover.STATE_CLOSING, cover.STATE_OPENING):
+                if (
+                    self.state.state
+                    in (
+                        cover.STATE_CLOSING,
+                        cover.STATE_OPENING,
+                    )
+                    or self.state.attributes.get(ATTR_ASSUMED_STATE)
```

Comment:
What is setting assumed_state to true?

(This all makes sense, just trying to make sure I understand how it works.)

Reply:
The integration for the cover, or customization of entity.

Reply:
Got it, thanks. I found the trait registration logic.  This all makes sense to me.